### DISCUSSION TITLE
nixOS support

### DISCUSSION THREAD

**Discussion by tcurdt** [coollabsio/coolify#1855] [Feature Requests]
I presume this will be still very niche and maybe not that easy to do - but it would be very nice to get coolify included in

https://search.nixos.org/packages

**Comment by tcurdt** [comment]
Looking at the script and the the things needed this might actually not be that hard at all:

- Install basic commands: curl wget git jq jc
- Docker Engine (24+)
- Configures proper logging for Docker Engine.
- Creates directory structure at /data/coolify for all the configuration files.
- Creates an SSH key for Coolify to be able to manage this server from itself at /data/coolify/ssh/keys/id.root@host.docker.internal.
- Install dockerized Coolify.

Is it possible to change the location of the data to something other than `/data`?

**Comment by Kirirou** [comment]
Would be great if someone could make NixOS Support.

There is an open issue package request on nixpkgs: 
- https://github.com/NixOS/nixpkgs/issues/303482
- https://github.com/NixOS/nixpkgs/issues/291589

**Reply by RayBB** [reply]
Those are closed now but there is one open for the coolify CLI!
https://github.com/NixOS/nixpkgs/pull/500295
